src/promptflow/promptflow/_sdk/_submitter/test_submitter.py

```diff
@@ -241,9 +242,14 @@ def init(
             # temp flow is generated, will use self.flow instead of self._origin_flow in the following context
             self._within_init_context = True
 
-            if self.flow.language == FlowLanguage.CSharp:
+            if self.flow.language != FlowLanguage.Python:
+                ExecutorProxyFactory().get_executor_proxy_cls(self.flow.language).generate_flow_metadata(
+                    flow_file=self.flow.path,
+                    working_dir=self.flow.code,
+                    dump=True,
+                )
                 # TODO: consider move this to Operations
-                CSharpExecutorProxy.generate_metadata(self.flow.path, self.flow.code)
+                CSharpExecutorProxy.generate_flow_metadata(self.flow.path, self.flow.code)
```

Comment:
why we call generate_flow_metadata twice?

Reply:
a mistake brought in a conflict resolution, let me remove it